shacl12-sparql/index.html

```diff
@@ -1206,7 +1206,113 @@ <h3>Validation with SPARQL-based Constraint Components</h3>
 				</p>
 			</section>
 		</section>
-		
+				
+		<section id="sparql-constraints-annotations">
+			<h2>Annotation Properties</h2>
+			<p>
+				This section extends the general <a href="#sparql-constraints-variables">mechanism</a>
+				to produce validation results using <a href="sparql-constraints">SPARQL-based constraints</a> or
+				<a href="sparql-constraint-components">constraint components</a>.
+			</p>
+			<p>
+				Implementations that support this feature make it possible to inject <dfn>annotation properties</dfn>
+				into the validation result nodes created for each <a>solution</a> produced by the <code>SELECT</code> queries of a
+				SPARQL-based <a>constraint</a> or <a>constraint component</a>.
+				Any such annotation property needs to be declared via a <a>value</a> of <code>sh:resultAnnotation</code> at
+				the <a>subject</a> of the <code>sh:select</code> or <code>sh:ask</code> <a>triple</a>.
+			</p>
+			<p class="syntax">
+				<span data-syntax-rule="resultAnnotation-nodeKind">The <a>values</a> of <code>sh:resultAnnotation</code> are
+				called <dfn data-lt="result annotation">result annotations</dfn> and are either <a>IRIs</a> or <a>blank nodes</a></span>.
+			</p>
+			<p>
+				<a>Result annotations</a> have the following properties:
+			</p>
+			<table class="term-table">
+				<tr>
+					<th>Property</th>
+					<th>Summary and Syntax Rules</th>
+				</tr>
+				<tr>
+					<td><code>sh:annotationProperty</code></td>
+					<td>
+						The property that shall be set.
+						<span data-syntax-rule="annotationProperty">Each <a>result annotation</a> has exactly one <a>value</a>
+						for the property <code>sh:annotationProperty</code> and this value is an <a>IRI</a>.</span>
+					</td>
+				</tr>
+				<tr>
+					<td><code>sh:annotationVarName</code></td>
+					<td>
+						The name of the SPARQL variable to take the annotation values from.
+						<span data-syntax-rule="annotationVarName">Each <a>result annotation</a> has at most 1 <a>value</a>
+						for the property <code>sh:annotationVarName</code> and this <a>value</a> is <a>literal</a> with
+						<a>datatype</a> <code>xsd:string</code>.</span>
+					</td>
+				</tr>
+				<tr>
+					<td><code>sh:annotationValue</code></td>
+					<td>
+						Constant <a>RDF terms</a> that shall be used as default values.
```

Comment:
For the sake of cross-linking consistency in `shacl-shacl.ttl`, can a `data-syntax-rule` `span` be added into this line?

Reply:
Done [in 377](https://github.com/w3c/data-shapes/pull/377/commits/e68d239e2e35644fef71429dfcd5e460f66fc0c0).